<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDouble</class>
 <widget class="QWidget" name="WidgetDouble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>573</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom du paramètre :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="alias"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Libellé (avant) :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="libBefore"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Libellé (après, unité) :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="libAfter"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Nombre de décimales :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="nbDec">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Valeur minimale :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="min">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Valeur maximale :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="max">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Valeur par défaut :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="defaultValue">
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Multiplicateur (pour affichage) :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="mult">
     <property name="minimum">
      <double>-999999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description du paramètre :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
